examples/byte-type/byte_type.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE introduces the Ballerina byte type.
+keywords:  ballerina, ballerina by example, bbe, bytes, byte, byte type
```

Comment:
Shall we add unsigned8 also? @SasinduDilshara @MaryamZi

Reply:
+1

Reply:
In that case we may need to refer to https://ballerina.io/learn/by-example/built-in-integer-subtypes/ as well.

Reply:
done